src/tools/scripts/deploy.sh

```diff
@@ -98,10 +98,30 @@ if [ "$(pgrep -f 'python main.py')" ]; then
   pkill -9 python main.py
 fi
 
-#Remove generated chapters (in case new one from other branch in there)
+#Remove generated chapters and e-books (in case new one from other branch in there)
 # Or with "true" to prevent failure if files don't exist
-echo "Removing Generated Chapters"
+echo "Removing Generated Chapters and e-books"
 rm -f ./templates/*/*/chapters/* || true
+rm -f ./static/pdfs/* || true
+
+# Download latest e-books from Github Action artifacts
+echo "Downloading latest e-books from Github Action artifacts"
+# Set your GitHub repository and workflow name
+REPO_OWNER=HTTPArchive
+REPO_NAME=almanac.httparchive.org
+WORKFLOW_ID=predeploy.yml
+# Set your Github personal access token
+export $(cat ../.env | xargs)
+#read -sp 'Enter your Github personal access token: ' GITHUB_TOKEN
```

Comment:
@tunetheweb please adjust to whatever is preferred option to pass GITHUB_TOKEN value.
Here I tested with `.env` file and user input.